src/components/TorrentDetail/Tabs/Info.vue

```diff
@@ -139,25 +139,19 @@
           <td :class="commonStyle">
             {{ $t('modals.detail.pageInfo.trackers') }}
           </td>
-          <td>
-            {{ torrent.tracker }}
-          </td>
+          <td v-html="$options.filters.transformUrlToHref(torrent.tracker)" />
         </tr>
         <tr v-if="createdBy">
           <td :class="commonStyle">
             {{ $t('modals.detail.pageInfo.createdBy') }}
           </td>
-          <td>
-            {{ createdBy }}
-          </td>
+          <td v-html="$options.filters.transformUrlToHref(createdBy)" />
         </tr>
         <tr v-if="comment">
           <td :class="commonStyle">
             {{ $t('torrent.comments') | titleCase }}
           </td>
-          <td>
-            {{ comment }}
```

Comment:
Seems like I forgot to press save 😭 
Other than that everything looks great :)

Reply:
I totally agree with you that it looks much cleaner. But the problem is that using this way, filter always returns _string_, and shows all html elements as string. So output from filter would look like "<a href='smth.else'>smth.else</a>". That's why I have used v-html directive - so it will insert content as plain HTML :)